src/components/Authentication/__tests__/LoginComponent.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Enzyme, { mount, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+
+
+import LoginComponent from '../Login/LoginComponent';
+import LoginContainer from '../Login/LoginContainer';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+
+describe('<LoginContainer />', () => {
+  it('should match the snapshot', () => {
+    const component = shallow(<LoginContainer />);
+    expect(component).toMatchSnapshot();
+  });
+});
+
+describe('Login component form', () => {
+  const store = mockStore({});
+  let wrapper;
+  const onChange = jest.fn();
```

Comment:
While there is nothing wrong with this it's not necessary. Inside the props object, you can just have say `onChange` as a key and `jest.fn()` as its value.

Reply:
Actually, I have tried it but it does not work so I think this is the best way to implement it as for now 
<img width="296" alt="screenshot 2019-01-21 at 21 38 26" src="https://user-images.githubusercontent.com/36000749/51493047-06b81180-1dc5-11e9-8665-91a3e534255a.png">

Reply:
Lemme try something first then give you feedback